<launch> 
    <node name="g4t1" pkg="component" type="g4t1" output="screen" />

    <param name="frequency" value="1" /> <!-- 1 Hz  -->

    <param name="connect" value="true" type="bool" />

    <param name="persist" value="false" type="bool" />
    <param name="path" value="g4t1_output.csv" />
    <param name="session" value="25" type="int" />

</launch>